aio_pika/patterns/master.py

```diff
@@ -99,7 +99,7 @@ def __init__(
         :param channel: Initialized instance of :class:`aio_pika.Channel`
         """
         self.channel: AbstractChannel = channel
-        self.loop: asyncio.AbstractEventLoop = self.channel.loop
```

Comment:
Why not to continue to use existing loop value?

Reply:
Yeah it's will conflicts with future releases